.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+---
+#based on https://raw.githubusercontent.com/ckan/ckanext-scheming/master/.github/workflows/test.yml
+# alternative https://github.com/ckan/ckan/blob/master/contrib/cookiecutter/ckan_extension/%7B%7Bcookiecutter.project%7D%7D/.github/workflows/test.yml
+name: Tests
+on:
+  push:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+
+    name: OpenData theme build
+    runs-on: ubuntu-latest
+    container: integratedexperts/ci-builder
```

Comment:
thats different, so what does this container do differently?

Reply:
It's the container that Salsa set up. It has Ahoy v2 and the necessary Docker-related tools installed.